packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -328,6 +328,10 @@ export const SharedTreeFormatVersion = {
 
 	/**
 	 * Requires \@fluidframework/tree \>= 2.0.0.
+	 *
+	 * @deprecated - FF does not currently plan on supporting this format long-term.
+	 * Do not write production documents using this format, as they may not be loadable in the future.
+	 *
```

Comment:
My understanding is that v2 is as deprecated as v1 and only v3 is supported. If that's the case then this should go into main regardless of what happens to this PR/effort.